packages/jest-serializer/src/__tests__/index.test.ts

```diff
@@ -18,7 +18,9 @@ import serializer from '..';
 
 const v8s = [
   {
+    // @ts-ignore - Node 8+ only
```

Comment:
Instead of disabling all those node 8 checks, you should mybe patch the node types.

```
declare namespace NodeJS {
  interface v8 {
    // ...
  }
}
```

Reply:
Thanks! Added `global.d.ts` into `includes`, hope it's ok? I'm new to TS :P